nms/app/fbcnms-projects/magmalte/app/components/layout/CardTitleRow.js

```diff
@@ -64,8 +64,10 @@ export const CardTitleFilterRow = (props: CardTitleFilterRowProps) => {
   return (
     <Grid container alignItems="center" className={classes.cardTitleRow}>
       <Grid item xs>
-        {Icon ? <Icon className={classes.cardTitleIcon} /> : null}
-        <Text variant="body1">{props.label}</Text>
+        <Grid container alignItems="center">
+          {Icon ? <Icon className={classes.cardTitleIcon} /> : null}
+          <Text variant="body1">{props.label}</Text>
+        </Grid>
```

Comment:
This just centers aligns the title/icon portion of the filtered Card Title components.